_api-reference/index-apis/create-index.md

```diff
@@ -113,4 +111,42 @@ index.routing.allocation.enable | Specifies options for the index’s shard allo
 index.routing.rebalance.enable | Enables shard rebalancing for the index. Available options are `all` (allow rebalancing for all shards), `primaries` (allow rebalancing only for primary shards), `replicas` (allow rebalancing only for replicas), and `none` (do not allow rebalancing). Default is `all`.
 index.gc_deletes | Amount of time to retain a deleted document's version number. Default is `60s`.
 index.default_pipeline | The default ingest node pipeline for the index. If the default pipeline is set and the pipeline does not exist, then index requests fail. The pipeline name `_none` specifies that the index does not have an ingest pipeline.
-index.final_pipeline | The final ingest node pipeline for the index. If the final pipeline is set and the pipeline does not exist, then index requests fail. The pipeline name `_none` specifies that the index does not have an ingest pipeline.
\ No newline at end of file
+index.final_pipeline | The final ingest node pipeline for the index. If the final pipeline is set and the pipeline does not exist, then index requests fail. The pipeline name `_none` specifies that the index does not have an ingest pipeline.
+
+#### Index codec settings 
```

Comment:
As discussed offline, we will add an additional page to explain the index.codec settings around the time of release. 
This information might be redundant post the addition of that page, how do you suggest we should proceed @hdhalter

Reply:
Calling it out here: We might need to update the [documentation](https://opensearch.org/docs/latest/tuning-your-cluster/availability-and-recovery/snapshots/snapshot-restore/#conflicts-and-compatibility) related to snapshot, and we can check if we can add something over here at [best practices for performance](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/bp.html#bp-perf)

Reply:
We need to add more information about indexing, in general, and will add more information to support these options after the release.

Reply:
Can you please provide me an endpoint so I can test the code samples on Monday? Thanks!

Reply:
@sarthakaggarwal97 - The documentation related to snapshots has been updated. The link above to 'best practices' is part of the Service documentation and needs to be handled by @chatnish.

Reply:
> Can you please provide me an endpoint so I can test the code samples on Monday? Thanks!

Do you want a domain where you can test these code samples?